data/tags.json

```diff
@@ -336,10 +338,22 @@
     "content": ["https://tryitands.ee/"]
   },
   { "name": "vim4newcomers",
-    "aliases": [],
+    "aliases": ["vim"],
     "content": [
-      "The best way to get started with vim is using `vimtutor`, which is pre-included with vim / neovim!",
-      "If you're looking for some pre-configured framework with sane defaults and plugins, [LunarVim](https://github.com/LunarVim/LunarVim) might be for you!"
+      "**The best way to learn vim is using tutor, which is pre-included with vim/neovim!**",
+      "- vim: `vimtutor`",
+      "- nvim: `nvim :Tutor`",
+      "- helix: `hx --tutor`",
```

Comment:
Surprised to see this here. Is helix that big? Doesn't feel like it comes up much.

Reply:
ive seen a few people in chat [try it](https://www.reddit.com/r/termux/comments/1abcter/how_to_setup_helix_in_termux/)
some might prefer it because rust 🚀🚀

including can be helpful even for previous vim users since the keybinds are slightly different

Reply:
I hope you realize "its written in Rust" is not a reason to recommend it.

Reply:
It definitely exists and has users, but I'm trying to avoid having any part of this file turn into a laundry list. We all have favorite projects; the tags will get bloated if we can all add them, and it's not at all clear that it's helpful or responsible to redirect someone looking for one of the big two to a vastly smaller project. I don't think it makes the cut here.